src/contracts/ownership/InitializedOwnable.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.7.5;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Ownable contract with custom intial owner
+ * @author Gnosis Developers
+ * @dev A contract extending Openzeppelin's Ownable contract that allows to
+ * specify the initial owner in the contructor instead of using the message
+ * sender.
+ */
+abstract contract InitializedOwnable is Ownable {
+    /**
+     * @dev Initializes the contract setting the input address as the initial
+     * owner.
+     */
+    constructor(address initialOwner) {
```

Comment:
nit: Maybe add a test case where deployer and owner are different to exhibit the specific use case?

Reply:
This should already be covered as the `deployer != owner`:

https://github.com/gnosis/oba-contracts/blob/84cf21da6db48dac522a0fccc24ba7c0e3d85199/test/GPv2AllowListAuthenticator.test.ts#L6

Reply:
I think the test case still says "should set deployer as owner"

Reply:
Ah good catch, it should be fixed then. I added a suggestion here https://github.com/gnosis/oba-contracts/pull/186#discussion_r530881285

Reply:
Good point, I updated the text and added a test to check that the deployer is not the owner for good measure.